config/config.js

```diff
@@ -17,5 +17,6 @@ module.exports = {
   hotjarId: process.env.HOTJAR_ANALYTICS_TRACKING_ID,
   googleAnalyticsId: process.env.GOOGLE_ANALYTICS_TRACKING_ID,
   webtrendsId: process.env.WEBTRENDS_ANALYTICS_TRACKING_ID,
-  cacheTimeoutSeconds: process.env.CACHE_TIMEOUT_SECONDS || 0
+  cacheTimeoutSeconds: process.env.CACHE_TIMEOUT_SECONDS || 0,
+  choicesUrl: process.env.CHOICES_URL || 'https://www.nhs.uk'
```

Comment:
Why is the default `https` and not `http` given Choices doesn't work on the former?

Reply:
Since this comment was made I've confirmed the use of `https` should be default.